logs/src/index.php

```diff
@@ -237,7 +240,7 @@
 $query .= " ORDER BY ctime DESC, host_name ASC, log_id DESC, service_description ASC";
 $query .= " LIMIT " . ($page * $preferences['entries']) . "," . $preferences['entries'];
 $res = $dbb->query($query);
-$nbRows = $res->rowCount();
+$nbRows = $dbb->numberRows();
```

Comment:
$nbRows =  $dbb->query("SELECT FOUND_ROWS()")->fetchColumn();
```suggestion
$nbRows = $dbb->numberRows();
```